client/cypress/integration/visualizations/chart_spec.js

```diff
@@ -99,19 +98,23 @@ function assertAxesAndAddLabels(xaxisLabel, yaxisLabel) {
   cy.getByTestId("Chart.LeftYAxis.Name")
     .clear()
     .type(yaxisLabel);
+
+  cy.getByTestId("VisualizationEditor.Tabs.General").click();
 }
 
-function createDashboardWithCharts(title, charts, widgetsAssertionFn = () => {}) {
-  return cy.createDashboard(title).then(dashboard => {
+function createDashboardWithCharts(title, chartGetters, widgetsAssertionFn = () => {}) {
+  cy.createDashboard(title).then(dashboard => {
     const dashboardUrl = `/dashboards/${dashboard.id}`;
-    return cy
-      .all(
-        charts.map((chart, i) => {
-          const position = { autoHeight: false, sizeY: 8, sizeX: 3, col: (i % 2) * 3 };
-          return () => cy.addWidget(dashboard.id, chart.id, { position });
-        })
-      )
-      .then(widgets => widgetsAssertionFn(widgets, dashboardUrl));
+
+    const widgets = [];
+    chartGetters.forEach((chartGetter, i) => {
+      const position = { autoHeight: false, sizeY: 8, sizeX: 3, col: (i % 2) * 3 };
+      cy.get(chartGetter).then(chart => {
+        widgets.push(cy.addWidget(dashboard.id, chart.id, { position }));
+      });
+    });
+
+    widgetsAssertionFn(widgets, dashboardUrl);
```

Comment:
For some reason, this is not running 🤪. @gabrieldutra can you give me a little insight?